tools/ossf_best_practices/silver_criteria.md

```diff
@@ -0,0 +1,321 @@
+
+_Possible answers: Met/Unmet_
+
+> The project MUST clearly define and publicly document the key roles in the project and their responsibilities, including any tasks those roles must perform. It MUST be clear who has which role(s), though this might not be documented in the same way. (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST be able to continue with minimal interruption if any one person dies, is incapacitated, or is otherwise unable or unwilling to continue support of the project. In particular, the project MUST be able to create and close issues, accept proposed changes, and release versions of software, within a week of confirmation of the loss of support from any one individual. This MAY be done by ensuring someone else has any necessary keys, passwords, and legal rights to continue the project. Individuals who run a FLOSS project MAY do this by providing keys in a lockbox and a will providing any needed legal rights (e.g., for DNS names).
+
+_Possible answers: Met/Unmet_
+
+> The project SHOULD have a "bus factor" of 2 or more. (URL required)
+
+_Possible answers: Met/Unmet_
+
+## Documentation
+
+> The project MUST have a documented roadmap that describes what the project intends to do and not do for at least the next year. (URL required)
+
+_Possible answers: Met/Unmet_
+
+The project MUST include documentation of the architecture (aka high-level design) of the software produced by the project. If the project does not produce software, select "not applicable" (N/A). (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+> The project MUST document what the user can and cannot expect in terms of security from the software produced by the project (its "security requirements"). (URL required)
+
+_Possible answers: Met/Unmet_
+
+> The project MUST provide a "quick start" guide for new users to help them quickly do something with the software. (URL required)
+
+_Possible answers: Met/Unmet/NA_
+
+
+The project MUST make an effort to keep the documentation consistent with the current version of the project results (including software produced by the project). Any known documentation defects making it inconsistent MUST be fixed. If the documentation is generally current, but erroneously includes some older information that is no longer true, just treat that as a defect, then track and fix as usual.
+
+_Possible answers: Met/Unmet/NA_
+
+
+The project repository front page and/or website MUST identify and hyperlink to any achievements, including this best practices badge, within 48 hours of public recognition that the achievement has been attained. (URL required) 
+
+_Possible answers: Met/Unmet_
+
+
+## Accessibility and internationalization
+
+> The project (both project sites and project results) SHOULD follow accessibility best practices so that persons with disabilities can still participate in the project and use the project results where it is reasonable to do so.
+
+_Possible answers: Met/Unmet/NA_
```

Comment:
@ovflowd is this requirement met with the new website (accessibility)?

Reply:
![Captura de pantalla 2023-05-17 a las 15 04 34](https://github.com/nodejs/security-wg/assets/5110813/ecbeb660-f763-457e-96ca-17388d045711)

Seems like criteria is met @marco-ippolito 🎉

Reply:
@UlisesGascon could you try that test on https://nodejs.dev?

Yes, this is one of the requirements for the new website to meet a11y standards. But be aware that the current template will change to the one on nodejs.dev

Reply:
Sure thing @ovflowd!

Here is the report:
<img width="1452" alt="Captura de pantalla 2023-05-18 a las 10 53 19" src="https://github.com/nodejs/security-wg/assets/5110813/829cfbf1-4dc1-4fd8-8f7e-65316324dfed">

Seems like only one page can be scanned as the tool didn't find the `robots.txt` or internal links 🤔 

![Captura de pantalla 2023-05-18 a las 10 54 32](https://github.com/nodejs/security-wg/assets/5110813/0a54c6b4-97b5-4cd0-a440-987a4c77f11b)

Maybe is a bug in the tool, because the links are included


<img width="1208" alt="Captura de pantalla 2023-05-18 a las 10 58 05" src="https://github.com/nodejs/security-wg/assets/5110813/ebec9390-a792-41f5-bd66-13958efd0038">


But in any case, we have a very good scoring 🚀

Reply:
resolved in a31b29567b4a5eca39e4621557640ed848a8417a